static/src/javascripts/__flow__/types/ab-tests.js

```diff
@@ -49,10 +72,8 @@ declare type DeploymentRules = 'AlwaysAsk' | MaxViews
 declare type EpicABTest = AcquisitionsABTest & {
     campaignPrefix: string,
     useLocalViewLog: boolean,
-    overrideCanRun: boolean,
     onlyShowToExistingSupporters: boolean,
     pageCheck: (page: Object) => boolean,
-    locations: $ReadOnlyArray<string>,
     useTargetingTool: boolean,
     insertEvent: string,
```

Comment:
shouldn't `EpicABTest` now have variants of type `EpicVariant[]`?

Reply:
I just tried adding `variants: $ReadOnlyArray<EpicVariant>` to `EpicABTest`
It fails flow's typechecking in `makeEpicABTest` where I initialise it as `variants: []`, not sure why:
```
Cannot assign object literal to `test` because:
 - property `buttonTemplate` is missing in `Variant` [1] but exists in object type
 [2] in array element of property
   `variants`.
```

Reply:
And actually, maybe the `EpicVariant` type is unnecessary.
All these fields could just stay in `InitEpicABTestVariant`, and when we construct the `Variant` we reference them when they're used, e.g. `initVariant.tagIds`

Reply:
Actually, it is useful to be able to pass `EpicVariant` into functions.

Reply:
I may revisit this later. For now I'm going merge and unblock those other tasks